webdriver-spec.html

```diff
@@ -1970,9 +1969,15 @@
   <td>Defines the password used when <a>authenticating</a> with a <a>SOCKS proxy</a>,
    when the <a><code>proxyType</code></a> is "<code>manual</code>",
    and is not returned with <a>capabilities</a> to the <a>local end</a>.
-  <td>
+  <td>Any <a>string</a>
  </tr>
 </table>
+
+<p>A <dfn>valid proxy configuration object</dfn> is a
```

Comment:
We can drop `valid` since a proxy configuration object is either valid or its not a proxy configuration object